### ISSUE TITLE
SeatAndEat : Make Hero Section Responsive In The Menu Page

### ISSUE THREAD

**Issue by Rajarshi12321** [DSCKGEC/SeatAndEat#14]
- Make the Hero section responsive the Menu section.

**Comment by ahmedbilal-2711** [comment]
Please assign this to me

**Comment by ahmedbilal-2711** [comment]
Shouldn't it be in the Home Page?

**Comment by Rajarshi12321** [comment]
It actually refers to the carousel of the Menu page.

**Comment by Rajarshi12321** [comment]
I have assigned this issue to you as per your request. Feel free to reach out with your queries.

**Comment by ahmedbilal-2711** [comment]
The images on the right are causing problem. because their position is set according to left, top, bottom and right. Can we change the design a bit ?

**Comment by Rajarshi12321** [comment]
Yes ,you can enhance it according to you. You can share screenshots when you are done with that.

**Comment by ahmedbilal-2711** [comment]
Okay

**Comment by ahmedbilal-2711** [comment]
Please check it

**Comment by ahmedbilal-2711** [comment]
> Yes ,you can enhance it according to you. You can share screenshots when you are done with that.

Any updates?

**Comment by Rajarshi12321** [comment]
You are guided in the pr section to follow the ksoc guidelines and to share screenshots of your work.

**Comment by ahmedbilal-2711** [comment]
Is PR section Pull Request section?
Sorry I'm new to this all.

**Comment by Rupam-Matabber** [comment]
@ahmedbilal-2711 yes the PR section is the Pull request section.

- Solve this issue and make a new PR. While making the new PR the template will automatically come. Just edit it and attach the screenshots of your work.
